src/EnergyPlus/HeatingCoils.cc

```diff
@@ -475,7 +475,7 @@ namespace HeatingCoils {
             }
 
             heatingCoil.HeatingCoilType = "Heating";
-            heatingCoil.HeatingCoilModel = "ElectricMultiStage";
+            heatingCoil.HeatingCoilModel = "Electric:MultiStage";
```

Comment:
This results in a `CompType` of `Coil:Heating:ElectricMultistage` in the sizing routines which leads to information in the CoilSelectionReport object to be ignored because that is not a valid coil type name.